### DISCUSSION TITLE
it is not possible to paginate a view

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2235] [General]
![image](https://user-images.githubusercontent.com/3023526/128011154-b9a87767-e4a9-4baf-a3fb-bfbfdaf96ef8.png)
while without the view:
![image](https://user-images.githubusercontent.com/3023526/128011219-8f4bc6e9-31ab-419c-bfad-d251621c3fb7.png)

(I will filter on the table and not on the view, it is just to let you know)

**Comment by KernOil** [comment]
Hi, @oswaldB  views are not supported yet other than viewing them. I am waiting on the views functionality as well.
